bin/yb-ctl

```diff
@@ -1054,16 +1074,31 @@ class ClusterControl:
                 ("PID", pid),
                 ("admin service", "http://" + get_address("http"))
             ]
-
             if daemon_id.is_tserver():
+                ycql_port = self.options.base_ports[daemon_id.daemon_type]["ycql_rpc"]
+                yedis_port = self.options.base_ports[daemon_id.daemon_type]["yedis_rpc"]
                 info_list.extend([
-                    ("cql service", get_address("cql_rpc")),
-                    ("redis service", get_address("redis_rpc")),
+                    ("ycql service", "cqlsh {} {}".format(
```

Comment:
It's better to print out the full path (./bin/cqlsh) instead of just binary name (cqlsh).